versioned_docs/version-1.3.0/use-cases/distributed-mongodb-for-multicloud.mdx

```diff
@@ -0,0 +1,1069 @@
+  
+   Example
+   ```
+   kubectl -n mongodb-operator get sa | grep -E '^(mongo)'
+   ```
+
+   Example Output
+   ```
+   mongodb-enterprise-appdb           1         11m
+   mongodb-enterprise-database-pods   1         11m
+   mongodb-enterprise-operator        1         11m
+   mongodb-enterprise-ops-manager     1         11m
+   ```
+
+10. Verify if the Kubernetes Operator is installed correctly using the following command: 
+
+    ```
+    kubectl describe deployments mongodb-enterprise-operator -n "mongodb-operator"
+    ```
+
+### Step 6: Deploy the MongoDB Ops Manager
+
+  :::info
+  To know more, see [mastering MongoDB Ops Manager](https://www.mongodb.com/developer/products/connectors/mastering-ops-manager/).
+  The master must contain the MongoDB Enterprise Operator  deployed.
+  :::
+
+ 1. Switch the kubernetes context to your MongoDB master cluster using the following command:
+
+     ```
+     kubectx
+     ```
+   
+     ```
+     k8s-cluster-1
+     ```
+
+  2. Make sure that the operator is in the running state.
+
+     Example
+     ```
+     kubectl get po -n mongodb-operator
+     ```
+
+     Example Output
+     ```
+     NAME                                           READY   STATUS    RESTARTS   AGE
+     mongodb-enterprise-operator-68cb5dd658-v2wrf   1/1     Running   0          22m
+     ```
+
+2. To access the Operator Manager user interface after installation, create a 
+   secret containing the username and password on the master Kubernetes cluster using the following command:
+
+
+   ```
+   kubectl -n mongodb-operator create secret generic om-admin-secret \
+     --from-literal=Username="richie@aveshasystems.com" \
+     --from-literal=Password="avesha@2023" \
+     --from-literal=FirstName="Ops" \
+     --from-literal=LastName="Manager"
+   ```
+
+3. Deploy the Ops Manager using the Ops managers Custom Resource, creating MongoDBOpsManager object, using the following manifest:
+
+   ```yaml
+   kubectl apply -f 
+   apiVersion: mongodb.com/v1
+   kind: MongoDBOpsManager
+   metadata:
+     name: ops-manager
+     namespace: mongodb-operator
+   spec:
+     version: 6.0.5
+     # the name of the secret containing admin user credentials.
+     adminCredentials: om-admin-secret
+     externalConnectivity:
+       type: LoadBalancer
+     configuration:
+       mms.ignoreInitialUiSetup: "true"
+       automation.versions.source: mongodb
+       mms.adminEmailAddr: richie@aveshasystems.com
+       mms.fromEmailAddr: richie@aveshasystems.com
+       mms.replyToEmailAddr: richie@aveshasystems.com
+       mms.mail.hostname: aveshasystems.com
+       mms.mail.port: "465"
+       mms.mail.ssl: "false"
+       mms.mail.transport: smtp
+     # the Replica Set backing Ops Manager.
+     applicationDatabase:
+       members: 3
+       version: 5.0.5-ent
+   ```
+
+4. Verify Ops Manager and Ops Manager MongoDB application database pods are running.
```

Comment:
In the ENT use case, I've rephrased this sentence as - 

`Verify the Ops Manager and check its MongoDB application database pods using the following command`